5.5/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+ADD mysql /opt/openshift/
+ENV PATH /opt/openshift/bin:$PATH
+
+WORKDIR /opt/openshift
+RUN create_user.sh
```

Comment:
I tend to agree with @mfojtik, having it in Dockerfile is ugly, IMO. I agree that script ending up in the image is not ideal, but it's just around 135 bytes, so I don't think we need to cut corners there.

I would add arguments for UID, GID and name.